scripts/src/models/users.ts

```diff
@@ -0,0 +1,139 @@
+import { Column, Entity, PrimaryColumn } from "typeorm";
+
+import { Model, Register } from "./index";
+
+@Entity()
+@Register
+export class User extends Model {
+	@PrimaryColumn({ name: "id" })
+	private _id: string;
+
+	@Column({ name: "app_id" })
+	private _appId: string;
+
+	@Column({ name: "app_user_id" })
+	private _appUserId: string;
+
+	@Column({ name: "created_date" })
+	private _createdDate: Date;
+
+	@Column({ name: "activated_date" })
+	private _activatedDate: Date;
```

Comment:
reminder: yohay suggested to have a permissions object that each token encapsulates and a user has to allow permissions per token.